xml/deployment_cifs.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+ xml:id="cha.ses.cifs">
+<!-- ============================================================== -->
+ <title>Export &cephfs; via Samba</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer>tbazant@suse.com</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline></dm:deadline>
+   <dm:priority></dm:priority>
+   <dm:translation></dm:translation>
+   <dm:languages></dm:languages>
+   <dm:release>SES4</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+  This section describes how to export &cephfs; via Samba share. Samba shares
+  can be used with Windows* clients.
+ </para>
+ <warning>
+  <title>Technology Preview</title>
+  <para>
+   As of &storage; 5, <literal>CIFS Export</literal> is considered a technology
+   preview and is not supported.
+  </para>
+ </warning>
+ <sect1 xml:id="sec.ses.cifs.example">
+  <title>Example Installation</title>
+  <para>
+   Exporting &cephfs; is a technology preview and not supported. To enable
+   CIFS, you have to manually install Samba on one cluster node and configure it.
+   Failover functionality can be provided with CTDB and the &sle; &hasi;.
+  </para>
+  <para>
+   First install the Samba daemon on one node:
+  </para>
+<screen>&prompt.root;<command>zypper</command> in samba</screen>
+  <para>
+   Then edit the <filename>/etc/smb/smb.conf</filename> and add the following
+   section:
+  </para>
+<screen>[<replaceable>SHARE_NAME</replaceable>]
+ceph:config_file = /etc/ceph/ceph.conf</screen>
+  <para>
+   Start and enable the Samba daemon:
+  </para>
+<screen>&prompt.root;<command>systemctl</command> start smb.service
+&prompt.root;<command>systemctl</command> enable smb.service</screen>
+ </sect1>
+ <sect1 xml:id="sec.ses.cifs.ha">
+  <title>High Availability Active-Passive Configuration</title>
+  <para>
+   This section provides an example of how to set up a two-node active-passive
+   configuration of Samba servers. The setup requires the &sle; &hasi;.
+   The 2 nodes are called &si.earth; and &si.mars;.
+  </para>
+  <para>
+   For details about &sle; &hasi;, see
+   <link xlink:href="https://www.suse.com/documentation/sle-ha-12/" />.
+  </para>
+  <para>
+   In this setup &si.earth; has the
+   IP address <systemitem class="ipaddress">192.168.1.1</systemitem> and
+   &si.mars; has the address
+   <systemitem class="ipaddress">192.168.1.2</systemitem>.
+  </para>
```

Comment:
I think this paragraph and the next paragraph could be a list like here: http://docserv.suse.de/documents/SLE-HA12-SP3/SLE-HA-install-quick/html/#sec.ha.inst.quick.usage-scenario

I would propose something like this:

--- 
The procedures in this document will lead to a minimal setup of a two-node cluster with the following properties:

* Two nodes: earth (192.168.1.1) and mars (192.168.1.2).
* Two floating, virtual IP addresses which allows clients to connect to the service no matter which physical node it is running on. <systemitem class="ipaddress">192.168.1.10</systemitem> is used for cluster administration with Hawk2 and <systemitem class="ipaddress">192.168.2.1</systemitem> is used exclusively for the CIFS exports. This makes it easier to apply security restrictions later.
* Related Samba services like CTDB, smb, and nmb.